### ISSUE TITLE
[WIP] Research infrastructure roles chapter

### ISSUE THREAD

**Issue by Arielle-Bennett** [alan-turing-institute/the-turing-way#1924]
## Summary

This is an issue to collect ideas for a chapter on Research infrastructure roles in supporting collaboration, best research practices and open, inclusive research. Expect that this chapter will grow quite large and there will be a lot of scope for updating with specific case studies covering a range of different flavours of community manager/project manager/research application manager. 

Community managers are a growing but as yet under-utilised research infrastructure role. Their roles vary widely but all around involved in collating and cohering a group of researchers and other stakeholders around a shared goal, project or interest. 

Everyone involved in supporting, scaffolding, groundskeeping, and programming of communities is invited to contribute their ideas and notes. The exact location of this section in the guide (and whether it would fit better elsewhere) is up for discussion. 

This chapter is being drafted in the community-manager-chapter branch https://github.com/alan-turing-institute/the-turing-way/tree/community-manager-chapter

Ideas are also being collected in the HackMD here: https://hackmd.io/@iXOo9P13RRq_GsyrM8bKsw/r1TROX-YO/edit 

### Discussion Points

Ideas for chapters/sections (more welcome!) 

- [ ] Community manager overview
- [ ] Different tools that community managers use
- [ ] Differences between community managers vs project/programme managers vs research application managers
- [ ] Subchapters on different community manager examples (case studies!) 
- [ ] Subchapter on the concept of community managers as groundskeepers/stewards 
- [ ] Community Manager resources 

### Resources

- CSCCE.org
- OLS

## Who can help?

* Anyone with experience of community management in the sciences/academic research area 

## Updates
@Arielle-Bennett has updated the chapter title to reflect the scope of the chapter and subchapters which we hope will eventually contain examples of different types of research infrastructure roles as well as the overview. 

- [ ] Write chapter outline
- [ ] Add material to the chapter
- [ ] Combine materials into a readable chapter
- [ ] Proofread
- [ ] Request reviews
- [ ] Address reviews
- [ ] Merge to master branch.

**Comment by lottycoupat** [comment]
Job description from Kirstie could be helpful here? High level notes I made from it:
Compassion, Steward and Collaborate as the three main actions by a Community manager. 
* Catalyse connections with all stakeholders
* Foster communities (integrating communities together) 
* Linking the right people up together (based on expertise and interests) 
* Active community member
* Communicating technical topics to non-technical colleagues and vice versa
* Encouraging high standards of reproducible, ethical, inclusive and collaborative data science

**Comment by lottycoupat** [comment]
Initial thoughts on Community Manager vs Project Manager:
* Community Manager much more people facing
* Ensuring inclusive environments
* Creating space to enable collaboration

Community Manager needs to know the project plans and help get them distributed to the community in a way that is accessible for all!

**Comment by malvikasharan** [comment]
@AidaMehonic has a lot of notes on this one. Tagging her here in case she would like to collaborate on the chapter.

**Comment by AidaMehonic** [comment]
Thanks @malvikasharan  - indeed, I am keen to collaborate on this chapter - especially that at some point we might want to write a RAM version of it too in our quest towards the ontology of new roles in reserach :)

**Comment by BatoolMM** [comment]
This is a very interesting chapter. I don't have experience in this but there's an interesting section about community manager (CMGR) in the [Community development handbook by GOV.UK](https://www.gov.uk/government/publications/community-development-handbook/community-development-handbook). I came across this while trying to figure out the structure of OSCSA. There are also a few interesting resources I came across in a 2-day building research software communities workshop that I attended two months ago. It was organised by SORSE and mainly targeted at community managers (from various communities such as the Carpentries, RSE, Netherlands eScience Center, ...) and there was a talk/discussion session by CSCCE. I'll post the google docs notes to you in slack, in case you find something helpful in it.
